src/main/java/mate/academy/spring/validator/EmailValidator.java

```diff
@@ -0,0 +1,18 @@
+package mate.academy.spring.validator;
+
+import java.util.regex.Pattern;
+import javax.validation.ConstraintValidator;
+import javax.validation.ConstraintValidatorContext;
+
+public class EmailValidator implements ConstraintValidator<Email, String> {
+    private static final String EMAIL_PATTERN = "^(?=.{1,64}@)[A-Za-z0-9_-]+(\\.[A-Za-z0-9_-]+)*@"
+            + "[^-][A-Za-z0-9-]+(\\.[A-Za-z0-9-]+)*(\\.[A-Za-z]{2,})$";
+
+    @Override
+    public boolean isValid(String email, ConstraintValidatorContext constraintValidatorContext) {
+        if (email.isEmpty()) {
```

Comment:
Add check for null email.
![image](https://user-images.githubusercontent.com/35394922/208751491-9a8a99b5-606c-4d3b-9c5c-606555532e34.png)